File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:

        myDict = {}
        
        for i,v in enumerate(nums):
            targ = target - v
            if targ in myDict:
                return [myDict[targ], i]
            myDict[v] = i
            

        

File: Data Structures & Algorithms/concatenation-of-array/submission-1.cs
public class Solution {
    public int[] GetConcatenation(int[] nums) {
        // int n = nums.Length;
        int[] ans = new int[2*nums.Length];

        for (int i = 0; i < nums.Length; i++)
        {
            ans[i] = ans[nums.Length+i] = nums[i];
        }

        return ans;
    }
}